pkg/apis/servicecatalog/fake/pointers.go

```diff
@@ -0,0 +1,5 @@
+package fake
+
+func boolPtr(b bool) *bool {
+	return &b
```

Comment:
Oh, those are hanging around.  We'll probably want to move them here naturally at some point.